ui/de-lib/src/main/java/org/iplantc/de/apps/client/presenter/categories/AppCategoriesPresenterImpl.java

```diff
@@ -164,28 +167,41 @@ public void onFailure(Throwable caught) {
             @Override
             public void onSuccess(List<AppCategory> result) {
                 filterCategories(result);
-                createWorkspaceTab(selectedAppCategory);
-                createHPCTab();
+                addCategoriesToWorkspaceTree(selectedAppCategory);
+                addCategoriesToHPCTree();
+                autoSelectHPCCategory();
```

Comment:
heh, I'm not so great at naming things in my own code. Sometimes it just takes a second set of 👀